ws-nextjs-app/utilities/pageRequests/getPageData.ts

```diff
@@ -20,13 +20,17 @@ const getPageData = async ({
   pageType,
 }: PageDataParams) => {
   const pathname = `${id}${rendererEnv ? `?renderer_env=${rendererEnv}` : ''}`;
+  const url = new URL(pathname, 'https://bbc.com');
+  const rendererEnvVal = url.searchParams.get('renderer_env');
+  const parsedPathName = `${id}${rendererEnvVal ? `?renderer_env=${rendererEnvVal}` : ''}`;
```

Comment:
Could we rename a couple of params, so that we can keep the API for `fetchDataFromBFF` clean? i.e. no need to set `pathname: parsedPathName,` below

```suggestion
  const path = `${id}${rendererEnv ? `?renderer_env=${rendererEnv}` : ''}`;
  const url = new URL(path, 'https://www.bbc.com');
  const rendererEnvironment = url.searchParams.get('renderer_env');
  const pathname = `${id}${rendererEnvironment ? `?renderer_env=${rendererEnvironment}` : ''}`;
```